src/dbFirebase.js

```diff
@@ -118,9 +118,12 @@ async function getUser(id) {
   return fbUser.exists ? fbUser.data() : null;
 }
 
-function onPhotosToModerate(fn) {
-  return firestore.collection('photos').where("moderated", "==", null ).onSnapshot((sn) => {
-    const docs = sn.docs.map(extractPhoto);
+function onCollectionToModerate(collection, query, fn) {
```

Comment:
not fully understand this. As we are not moderating the feedbacks...

Reply:
Would it be better to change the name of the function